hubmap_commons/type_client.py

```diff
@@ -297,19 +332,20 @@ def __init__(self, type_webservice_url: StringOrNone = None):
                     map[safe_nm] = k
             self.alt_name_map = map
 
-    def getAssayType(self, name: str) -> _AssayType:
+    def getAssayType(self, name: StrOrListStr) -> _AssayType:
         """
         Given an assay name, return the associated assay type.  If a deprecated
         alt-name is provided, the returned assay type will use the up-to-date
         name.
         """
-        if name not in self.examples:
-            if name in self.alt_name_map:
-                name = self.alt_name_map[name]
+        safe_name = name if isinstance(name, str) else tuple(name)
+        if safe_name not in self.examples:
+            if safe_name in self.alt_name_map:
+                safe_name = self.alt_name_map[safe_name]
             else:
                 raise RuntimeError(f'No such assay_type {name},'
                                    ' even as alternate name')
-        return _AssayType(self.examples[name])
+        return _AssayType(self.examples[safe_name])
```

Comment:
To complement the above:
```suggestion
        return _AssayType(self.examples[type_name])
```

Reply:
I'm not at all sure it's clearer, but it is not less clear so I'll accept it.